api/http/server.go

```diff
@@ -252,6 +254,11 @@ func (s *Server) HandleBzzGet(w http.ResponseWriter, r *http.Request) {
 				respondError(w, r, err.Error(), http.StatusUnauthorized)
 				return
 			}
+			if isRecoveryAttemptError(err) {
+				w.Header().Set("WWW-Authenticate", fmt.Sprintf("Basic realm=%q", uri.Address().String()))
+				respondError(w, r, err.Error(), http.StatusPaymentRequired)
```

Comment:
no i meant 420 not  402, you can just construct the response as:

    respondError(w, r, err.Error(), 420)

Reply:
My bad, I was confused it did not make sense.
Changed